bin/collator/src/command.rs

```diff
@@ -895,6 +903,28 @@ pub fn run() -> Result<()> {
                     error!("{}", err_msg);
                     Err(err_msg.into())
                 }
+
+                #[cfg(not(feature = "evm-tracing"))]
+                if config.chain_spec.is_astar() {
+                    start_astar_node(config, polkadot_config, collator_options, para_id, cli.enable_evm_rpc)
+                        .await
+                        .map(|r| r.0)
+                        .map_err(Into::into)
```

Comment:
This isn't clean nor is it scalable - why not instead introduce a new struct/type, like `AdditionalConfiguration` which would aggregate all additional config types?

That way you keep a single entry point for the `start_xyz_node` function. If we need to add additional ones in the future, how will that look like otherwise?

Reply:
Good point for refactoring the service.